CHANGELOG.md

```diff
@@ -46,6 +46,8 @@ Ref: https://keepachangelog.com/en/1.0.0/
 
 ## [Unreleased]
 
+## [v4.1.0](https://github.com/umee-network/umee/releases/tag/v4.0.1) - 2023-02-15
```

Comment:
Can you check the tag once

Reply:
oops - changed it (and the link will start working once we tag for real)